content/developer/reference/backend/security.rst

```diff
@@ -418,6 +418,40 @@ likely it is to break things.
     >> html_sanitize(code, strip_classes=True)
     '<p>Important Information</p>'
 
+**Markup**
+
+The library Markup is used in Odoo to escape and sanitize the content.
+When using Markup, the developer must ensure that the user content is passed correctly.
+
+The inside of the constructor of Markup is used for trusted content, you should never pass user-modifiable
+the content inside of the constructor of markup.
+
+.. code-block:: python
+
+    >>text = Markup('<strong> Your username is %s </strong>') % self.env.user.name
+    # We put our HTML inside of the constructor because it is trusted and fully under the control of the developer
+    # Since the user's name is passed by the operator '%', it will be escaped and any HTML code present will not be interpreted by the browser.
```

Comment:
Question, does `Markup` supports new-format-style too? In which case I would rather read a `.format` instead of a `%` but that's kinda nitpicking and personal preferences